CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Current
+### Added
+- Bundled latest version of try-convert which adds support for WPF and Unit Test projects written with Visual Basic #403
```

Comment:
Could you reword this to just add support for WPF and unit test projects written with VB?